packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -76,25 +74,33 @@ export function WithdrawalConfirmationDialog(
   const [checkbox1Checked, setCheckbox1Checked] = useState(false)
   const [checkbox2Checked, setCheckbox2Checked] = useState(false)
 
+  const { isArbitrumOne, isOrbitChain } = isNetwork(l2.network.id)
+  const baseChainId = isOrbitChain
+    ? (parentChains[l1.network.id] as L2Network)?.partnerChainID ?? mainnet.id
+    : l1.network.id
   const bothCheckboxesChecked = checkbox1Checked && checkbox2Checked
-  const confirmationSeconds =
-    getBlockTime(l1.network.id) * getConfirmPeriodBlocks(l2.network.id)
-  const confirmationDays = Math.ceil(confirmationSeconds / SECONDS_IN_DAY)
+
+  const { confirmationMins, confirmationHours, confirmationDays } =
+    getTxConfirmationTime({
+      baseChainId,
+      withdrawalFromChainId: l2.network.id
+    })
+
   let confirmationPeriod = ''
-  const confirmationHours = Math.ceil(confirmationSeconds / SECONDS_IN_HOUR)
 
   if (confirmationDays >= 2) {
     confirmationPeriod = `${confirmationDays} day${
       confirmationDays > 1 ? 's' : ''
     }`
+  } else if (confirmationHours >= 1) {
+    const roundedHours = Math.round(confirmationHours)
+    confirmationPeriod = `${roundedHours} hour${roundedHours > 1 ? 's' : ''}`
   } else {
-    confirmationPeriod = `${confirmationHours} hour${
-      confirmationHours > 1 ? 's' : ''
+    confirmationPeriod = `${confirmationMins} minute${
```

Comment:
If `confirmationMins` is 0 we should 'less than a minute' right?

Otherwise the UI would show '0 minute'

Reply:
`confirmationMins` is processed with `Math.ceil` so the case when it's 0 will never happen

tbh given the assertion time padding of 60 minutes, this case will never happen now but we're keeping it here